internal/server/data/migrations.go

```diff
@@ -466,19 +466,22 @@ func Migrate(db *gorm.DB) error {
 		// next one here
 	})
 
-	m.InitSchema(PreMigrate)
+	// TODO: why? isn't this already called by NewDB?
```

Comment:
We should capture this follow-up in an issue

Reply:
Created https://github.com/infrahq/infra/issues/2066 to track this